pkg/cdi/qualified-device.go

```diff
@@ -66,19 +66,6 @@ func ParseDevice(device string) (string, string, string) {
 	return parser.ParseDevice(device)
 }
 
-// ParseQualifier splits a device qualifier into vendor and class.
-// The syntax for a device qualifier is
-//
-//	"<vendor>/<class>"
-//
-// If parsing fails, an empty vendor and the class set to the
-// verbatim input is returned.
-//
-// Deprecated: use parser.ParseQualifier instead
```

Comment:
I think we should remove all deprecated functions with a single change.

Do we want to do that in a PR "before" this one, or as a follow-up?

Reply:
I'd prefer to do it in a follow-up PR.

Reply:
Yes, a follow-up is better. Please re-add the removed code in that case.